data/competencies.yml

```diff
@@ -1,16 +1,293 @@
+
+- id: cb1a4c9c-28dd-4d1d-938e-1f689fb91393
+  summary: Fixes or updates tests when prompted.
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: 43f6e213-b51e-45bd-bb37-a23a3e7d5457
+  summary: Checks whether code can be reused before writing new code, and generally identifies reuse when there is the opportunity
+  examples:
+    - Checks to use a python package in project
+    - Checks to use a node package in project
+    - Checks for the chance to integrate with proprietary software packages
+    - Checks Origami capabilities
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: cad93c51-dee9-4b10-8ccb-d2936dd95380
+  summary: Works to timeboxes when debugging
+  examples:
+    - Ensures they have a timebox when debugging.
+    - Communicates their progress through a timebox.
+    - Seeks help from other engineers before time runs
+    - Does not overshoot timebox without flagging it.
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: 0137fb68-807a-4971-8431-1b8bfd9a4bae
+  summary: Shadows during live incidents or joins debugging mob sessions.
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: 9098cc7f-a50a-48df-b84f-c4ee9fa47dbe
+  summary: Is able to explain why using CI/CD pipelines for automation is beneficial, and uses them for basic functions
+  examples:
+    - Restarts broken build.
+    - Is able to interpret CI output and articulate what is failing.
+    - Can fix simple problems or asks for help.
+  supportingUrls: []
+  description: null
+  level: junior
+  area: technical
+  domain: null
+
+- id: ae32dc05-3e53-47d2-a027-345eb42dffa3
+  summary: Occasionally uses monitoring tools to identify problems
+  examples:
+    - Occasionally uses Pingdom to identify problems
+    - Occasionally uses Grafana to identify problems.
+  supportingUrls: []
+  description: null
+  level: junior
   area: technical
   domain: null
 
+- id: 70ae7051-3aed-42a0-be39-6c8a381eeaaf
+  summary: Writes pull requests that are helpful both to the reviewer and those debugging the the code later
+  examples:
+    - PR descriptions that explain why the PR is needed
+    - Links relevant Jira ticket.
+    - Adds screenshots to show changes if applicable.
+    - Provides context that they learned when researching this PR.
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: 57c50830-f2ef-4c61-97fa-ec80b0624473
+  summary: Regularly reads peers' PRs and asks questions in order to learn.
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: 5d637365-7e38-432d-bcec-8652f06d8109
+  summary: Remembers to write tickets and bug reports to track the work expected of them, and can do so with guidance
```

Comment:
[Suggestion] Entry-level engineers might struggle to get experience of writing full tickets/bug reports. I'd expect work to be fairly well triaged before going to very junior people and some teams do an excellent job of preparing Jira tickets for all levels before they reach engineers. Would it be more reasonable to broaden this to writing, updating, or adding context to existing tickets or bug reports?

Reply:
That's fair. I've [corrected the text](https://github.com/niksilver/engineering-progression/commit/e3c1a25e0159dfc0c0074665fb39803fbf093f32) in another branch and rebase and squash it with all the others at the end.